src/Verification.php

```diff
@@ -44,15 +42,11 @@ public function __construct(VerificationCodesRepository $repo, GeneratorInterfac
     public function verify(string $action, string $redirectTo, \Closure $closure = null)
     {
         if ($this->shouldVerify($action)) {
-            $this->generateCodeAndSend($action);
+            $this->generateCodeAndSend($action, Container::getInstance()->make('request')->ip());
```

Comment:
chapem, ze `request()` nemas k disp. ale mas k disp fasau `Request::`, ved ju pouzivas na `URL` tak preco ine aj tu?

Reply:
Mas pravdu, uz to je zmenene.

Reply:
Nekazuje mi tie zmeny hmm. Asi si to uz mergol a potom si to fixoval v novom PR? Ale na tom PR nie som ako reviewer :-/